Cargo.toml

```diff
@@ -54,8 +55,10 @@ name = "crypto"
 harness = false
 
 [features]
-default = []
+default = ["secure"]
+secure = []
```

Comment:
Nit: we might want a comment here on how this is meant to control for accidental activations of `unsecure_schemes`.